src/ucs/sys/topo/gaudi/topo.c

```diff
@@ -0,0 +1,1911 @@
+/**
+* Copyright (C) Intel Corporation, 2025. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
```

Comment:
the code in this file seems gaudi-specific, can we move it to uct/gaudi (to the transport)?